### PR TITLE
Adding auto gen

### PR DESCRIPTION
**Pull request by TayGov**
# Description

This PR includes the following proposed change(s):

Add implementation of api

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/backend/efiling-reviewer-api/src/main/java/ca/bc/gov/open/jag/efilingreviewerapi/extract/DocumentsApiDelegateImpl.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.util.UUID;
+
+public class DocumentsApiDelegateImpl implements DocumentsApiDelegate {
+
+
+
+
```
Comment (alexjoybc) [inline src/backend/efiling-reviewer-api/src/main/java/ca/bc/gov/open/jag/efilingreviewerapi/extract/DocumentsApiDelegateImpl.java:13]:
```suggestion
```

File: src/backend/efiling-reviewer-api/src/main/java/ca/bc/gov/open/jag/efilingreviewerapi/extract/DocumentsApiDelegateImpl.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    @Override
+    public ResponseEntity<Object> extractDocumentFormData(UUID xTransactionId, String xDocumentType, MultipartFile file) {
+        return ResponseEntity.ok("Hello");
```
Comment (alexjoybc) [inline src/backend/efiling-reviewer-api/src/main/java/ca/bc/gov/open/jag/efilingreviewerapi/extract/DocumentsApiDelegateImpl.java:16]: Add a log ?

Comment (alexjoybc) [reply src/backend/efiling-reviewer-api/src/main/java/ca/bc/gov/open/jag/efilingreviewerapi/extract/DocumentsApiDelegateImpl.java:16]: question: are we gonna scan for virus ?

**Review by alexjoybc — COMMENTED** [review]
small stuff

File: src/backend/efiling-reviewer-api/src/main/java/ca/bc/gov/open/jag/efilingreviewerapi/extract/DocumentsApiDelegateImpl.java (hunk 2 above)
Comment (TayGov) [reply src/backend/efiling-reviewer-api/src/main/java/ca/bc/gov/open/jag/efilingreviewerapi/extract/DocumentsApiDelegateImpl.java:20]: Yes, should we add a task?

**Review by alexjoybc — APPROVED** [review]

**Pull request merged**